packages/docs/Next/Forms/Range.md

```
@@ -0,0 +1,43 @@
+# Range
+
+### Default
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)